polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -49,6 +54,8 @@ export interface FrameProps {
   showMobileNavigation?: boolean;
   /** Accepts a ref to the html anchor element you wish to focus when clicking the skip to content link */
   skipToContentTarget?: React.RefObject<HTMLAnchorElement>;
+  /** Internal use */
+  _internalFeatures?: FeaturesConfig;
```

Comment:
This seems like a code smell here creating "private" props. Could we avoid this if only using the AppProvider to control feature flags?